src/AddItemDialog.hpp

```diff
@@ -28,7 +28,9 @@
 #include <QDialog>
 #include "ui_additemdialog.h"
 #include <memory>
+#if QT_VERSION < 0x050000 && !defined(Q_MOC_RUN)
```

Comment:
This will never include for QT>=5.0

Maybe `#if !(QT_VERSION < 0x050000 && defined(Q_MOC_RUN))`
or `#if QT_VERSION >= 0x050000 || !defined(Q_MOC_RUN)`?

Reply:
Yep, I updated the commit. :)